addons/portal/views/portal_templates.xml

```diff
@@ -37,6 +37,25 @@
         </xpath>
     </template>
 
+    <template id="service_link_preview_front_end" inherit_id="portal.frontend_layout" primary="True">
+        <xpath expr="//head/meta[last()]" position="after">
+            <t t-if="record">
```

Comment:
`record` is almost impossible to grep ... and why would a generic portal template have a "record" ? Aren't you trying to make something generic while it does not seem necessary, based on this PR ?